samples/how-to-deal-with-import/CMakeLists.txt

```diff
@@ -0,0 +1,95 @@
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
```

Comment:
The sample is only about how to handle imports in wasm-c-api, it doesn't include the part of wamr apis, which has been demonstrated in `samples/native-lib`, how about renaming this folder to `wasm-c-api-imports` and adding README.md?

Reply:
Sure. Renaming the directory firstly, Upload README.md later.